ground/src/main/java/com/google/android/ground/ui/map/CameraPosition.kt

```diff
@@ -18,16 +18,16 @@ package com.google.android.ground.ui.map
 import com.google.android.ground.model.geometry.Coordinate
 
 data class CameraPosition(
-  val target: Coordinate,
+  val target: Coordinate? = null,
   val zoomLevel: Float? = null,
   val isAllowZoomOut: Boolean = false,
   val bounds: Bounds? = null
```

Comment:
I didn't realize we have bounds here; essentially (`target`, `zoomLevel`) and `bounds` model the same info. Also, `isAllowZoomOut` is an arg to an event, not to the current camera state. Can you file a bug (against me?) to clean this up and add a TODO here?

Reply:
https://github.com/google/ground-android/issues/1712